contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -477,6 +477,8 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
             address(proxyEOA),
             keccak256(Lib_EthUtils.getCode(address(proxyEOA)))
         );
+
+        _setAccountNonce(eoa, 0);
```

Comment:
EOA are expected to have a nonce of 0, but `_commitPendingAccount` sets the nonce to 1. So we reset it here.

Reply:
When are you planning on updating `_commitPendingAccount`?

Reply:
The problem is in other codepaths, `_commitPendingAccount` needs to set with 1.  One option is to add it as an input, not sure if necessary

Reply:
@kfichter actually, can we break this out into a separate PR?  @axchu is currently assigned to it, can reassign to you since solved here, but auditors want separate PR for this fix

Reply:
Sure thing, will remove the line from this PR and make a note.